Slambook/index.js

```diff
@@ -1,18 +1,25 @@
-let form;
-let allSlamArray=[];
-//Write code below to select the form element
-
-// Write your code above
-console.log(form,"form");
-
-// Add the evenlistner below
-
-//Write your code above
-
-function showSlamList(){
-//Update the function here
-
-// Write your code above this line
-}
-
+let form;
+let allSlamArray=[];
+//Write code below to select the form element
+form = document.querySelector ('.form');

```

Comment:
The console.log statement should be removed as it is unrelated to the task of updating the function